packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -75,14 +115,31 @@ const ShowHide = (props) => {
     }
   }
 
+  const onBlur = () => {
```

Comment:
What happens if there is a blur event and then the control regains focus?  Does that need to be handled?

Reply:
The specifics of this component is that the only purpose of the focus is to make the assistive technologies announce the text upon opening. For that we need the component to be only focus-able once, and when it looses focus - lose it forever. The only option for the component to become focus-able able again is if user closes and open it back.